stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/extensions/Message.kt

```diff
@@ -30,18 +30,25 @@ internal fun Message.updateUsers(users: Map<String, User>): Message =
 
 /**
  * Fills [Message.mentionedUsersIds] based on [Message.text] and [Channel.members].
+ *
+ * It combines the users found in the input with pre-set [Message.mentionedUsersIds], in case someone
```

Comment:
Let's mention it only searches for channel's members

Reply:
done!